sonar-kotlin-plugin/src/test/java/org/sonarsource/kotlin/checks/CheckTestWithNoSemantics.kt

```diff
@@ -24,6 +25,6 @@ abstract class CheckTestWithNoSemantics(
             this.fileName = sampleFileNoSemantics ?: "$checkName$NO_SEMANTICS_TEST_FILE_POSTFIX"
             this.classpath = this@CheckTestWithNoSemantics.classpath ?: emptyList()
             this.deps = this@CheckTestWithNoSemantics.dependencies ?: emptyList()
-        }.verifyNoIssue()
+        }.let { if (this.shouldReport) it.verify() else it.verifyNoIssue()}
```

Comment:
I think the formatting here isn't quire right. Also, I think it may aid readability a little bit to put the if-else statement on its own line.